packages/presentation/lib/widgets/card_screen.dart

```diff
@@ -47,14 +46,6 @@ class CardItem extends StatelessWidget {
     );
   }
 
-  Color? stringToColor(String color) {
-    if (color.contains("red")) {
-      return Colors.red;
-    } else {
-      return Colors.blue;
-    }
-  }
-
   String stringToString(String title) {
```

Comment:
what is this method for?

Reply:
I forgot it delete, it was a mapper
I create it when we just had started